README.md

```diff
@@ -19,33 +19,35 @@ From these Go sources, several files are generated:
 
 Generated files are created by a build script (see section [How to build](#how-to-build)).
 
-### Devfile 2.0.0 file format
+## Devfile 2.0.0 file format
 
 A Subset of this `DevWorkspace` API defines a structure (workspace template content), which is also at the core of the **Devfile 2.0** format specification.
-For more information about this, please look into the [Devfile support Readme](devfile-support/README.md)
+For more information about this, please look into the [Devfile support README]([devfile-support/README.md](https://github.com/devfile/registry-support/blob/main/README.md))
```

Comment:
This link is broken -- text is `Devfile support README` and link is to `[devfile-support/README.md](https://github.com/devfile/registry-support/blob/main/README.md)`

```suggestion
For more information about this, please look into the [Devfile support README](https://github.com/devfile/registry-support/blob/main/README.md)
```